README.md
Social Slider
========
JoomJunk Social Slider - A PHP module using CSS3 to show beautifully animated sliding icons 

Please see the changelog for detailed information about recent changes

See https://joomjunk.co.uk/products/social-slider.html for the latest stable version of the code. This github version will contain the latest version of the code - however this may still be a work in progress.

Some extra documentation can be found here https://joomjunk.co.uk/social-slider-docs.html


CHANGELOG.md
- Changelog

## Version 3.0.0
- Add support for Joomla 4

## Version 2.1.0
- Fixed default Twitter hex colour
- Fixed muddled icons on sorting feature
- Updated French language pack (thanks MATsxm)
- Update jQuery.sortable to v0.9.13
- Improve transition performance by using translateX
- Added Spanish language pack (thanks César Fernando)

## Version 2.0.2
- Fixed dark sprite icon order

## Version 2.0.1
- Fixed CSS spirte background positions

## Version 2.0.0
- Images compiled into sprite to improve performance
- Fixed AdBlocker not showing icons
- Updated Dutch language files

## Version 1.6.1
- Added Nofollow option
- Fixed update not running in 3.4 or later

## Version 1.6.0
- Added German languge pack
- Added Russian languge pack
- Fixed box-sizing issue from Bootstrap
- Fixed some CSS styling being overridden

## Version 1.5.7  (Important)
- Fixed version preventing updates from working

## Version 1.5.6
- Added French languge pack
- Added Dutch(Belgium) languge pack
- Styling tweaks/enhancements

## Version 1.5.5
- Slider container will no longer overlap other elements
- CSS enhancements
- Removed jQuery option and thus dropping support for IE8
- Removed support for Joomla 3.0 and 3.1

## Version 1.5.4
- Used more unique naming convertions to prevent overrides
- Removed unnecessary jQuery file for (Joomla 3.2)
- Updated jQuery to 1.11.1 (Joomla 2.5/3.1)

## Version 1.5.3
- Custom Slide text now translatable

## Version 1.5.2
- Added missing span to prevent text overlapping
- Added missing language string for parameter description
- Increased span width by 5px

## Version 1.5.1 (IMPORTANT BUG FIX FOR JOOMLA 3.1 USERS)
- Added background colour transition
- Fixed Joomla 3.1 module parameters page breaking

## Version 1.5.0
- Allow template overriding of CSS file
- Allow specification of the module from the bottom or the top of the page
- Script updates
- Add text colour parameter
- Fix bug in the sorting form field in J3.2

## Version 1.4.4 (IMPORTANT)
- Fixed missing closing bracket, breaking the site 

## Version 1.4.3
- Fixed overlapping text for long translated strings
- Major code optimization

## Version 1.4.2
- Fix for minor HTML markup validation error

## Version 1.4.1
- Code optimization of jQuery form field in Joomla 2.5 and lower
- jQuery form field displayed badly in Hathor

## Version 1.4.0
- Ability to order the position of the slides
- Improved code optimization of default.php file
- Moved assets folder into media folder

## Version 1.3.3
- Slide colour now a colour form field
- Undefined variable when module on right hand side of screen and jQuery enabled
- Language string changes RTL settings
- Google jQuery version updated to 1.10.2

## Version 1.3.2
- Ability to support RTL templates improved with dedicated option

## Version 1.3.1
- Bug fix for some users unable to open LastFM and Google+ tabs
- PHPCS fixes to Joomla Standards
- Max-width parameters added
- Add parameter for new tab or same page

## Version 1.3.0
- Added ability to use jQuery instead of CSS3 animations

## Version 1.2.3
- Added 5 custom slider tabs
- Removed unnecessary PHP tags 

## Version 1.2.2
- Added Vimeo icon

## Version 1.2.1
- Added additional black icons and ability to select which colour icons you want to display

## Version 1.2.0
- Added Github, Flickr and RSS icons
- Ability to disable module for mobile and tablet devices
- Removed Joomla 1.5 support

## Version 1.1.1
- Fixed issue with Last.fm always showing, even with parameter turned off

## Version 1.1.0
- Added in support for Steam, Last FM, Pinterest, SoundCloud and Tumbler

changelog.php
<?php
// No direct access
die('Restricted access');

?>

Changelog
------------

* -> Security Fix
# -> Bug Fix
$ -> Language fix or change
+ -> Addition
^ -> Change
- -> Removed
! -> Note

Version 2.1.0
+ Add support for Joomla 4

Version 2.1.0
# Fixed default Twitter hex colour
# Fixed muddled icons on sorting feature
$ Updated French language pack (thanks MATsxm)
^ Update jQuery.sortable to v0.9.13
^ Improve transition performance by using translateX
+ Added Spanish language pack (thanks César Fernando)

Version 2.0.2
# Fixed dark sprite icon order

Version 2.0.1
# Fixed CSS spirte background positions

Version 2.0.0
^ Images compiled into sprite to improve performance
# Fixed AdBlocker not showing icons
$ Updated Dutch language files

Version 1.6.1
+ Added Nofollow option
# Fixed update not running in 3.4 or later

Version 1.6.0
+ Added German languge pack
+ Added Russian languge pack
# Fixed box-sizing issue from Bootstrap
# Fixed some CSS styling being overridden

Version 1.5.7  (Important)
# Fixed version preventing updates from working

Version 1.5.6
+ Added French languge pack
+ Added Dutch(Belgium) languge pack
^ Styling tweaks/enhancements

Version 1.5.5
# Slider container will no longer overlap other elements
^ CSS enhancements
- Removed jQuery option and thus dropping support for IE8
- Removed support for Joomla 3.0 and 3.1

Version 1.5.4
+ Used more unique naming convertions to prevent overrides
- Removed unnecessary jQuery file for (Joomla 3.2)
^ Updated jQuery to 1.11.1 (Joomla 2.5/3.1)

Version 1.5.3
+ Custom Slide text now translatable

Version 1.5.2
# Added missing span to prevent text overlapping
# Added missing language string for parameter description
^ Increased span width by 5px

Version 1.5.1 (IMPORTANT BUG FIX FOR JOOMLA 3.1 USERS)
+ Added background colour transition
# Fixed Joomla 3.1 module parameters page breaking

Version 1.5.0
+ Allow template overriding of CSS file
^ Allow specification of the module from the bottom or the top of the page
^ Script updates
+ Add text colour parameter
# Fix bug in the sorting form field in J3.2

Version 1.4.4 (IMPORTANT)
# Fixed missing closing bracket, breaking the site 

Version 1.4.3
# Fixed overlapping text for long translated strings
^ Major code optimization

Version 1.4.2
# Fix for minor HTML markup validation error

Version 1.4.1
^ Code optimization of jQuery form field in Joomla 2.5 and lower
# jQuery form field displayed badly in Hathor

Version 1.4.0
+ Ability to order the position of the slides
^ Improved code optimization of default.php file
^ Moved assets folder into media folder

Version 1.3.3
^ Slide colour now a colour form field
# Undefined variable when module on right hand side of screen and jQuery enabled
$ Language string changes RTL settings
^ Google jQuery version updated to 1.10.2

Version 1.3.2
+ Ability to support RTL templates improved with dedicated option

Version 1.3.1
# Bug fix for some users unable to open LastFM and Google+ tabs
^ PHPCS fixes to Joomla Standards
+ Max-width parameters added
+ Add parameter for new tab or same page

Version 1.3.0
+ Added ability to use jQuery instead of CSS3 animations

Version 1.2.3
+ Added 5 custom slider tabs
^ Removed unnecessary PHP tags 

Version 1.2.2
+ Added Vimeo icon

Version 1.2.1
+ Added additional black icons and ability to select which colour icons you want to display

Version 1.2.0
+ Added Github, Flickr and RSS icons
+ Ability to disable module for mobile and tablet devices
- Removed Joomla 1.5 support

Version 1.1.1
# Fixed issue with Last.fm always showing, even with parameter turned off

Version 1.1.0
+ Added in support for Steam, Last FM, Pinterest, SoundCloud and Tumbler
